homeassistant/components/ld2410_ble/coordinator.py

```diff
@@ -26,15 +32,42 @@ def __init__(self, hass: HomeAssistant, ld2410_ble: LD2410BLE) -> None:
         ld2410_ble.register_callback(self._async_handle_update)
         ld2410_ble.register_disconnected_callback(self._async_handle_disconnect)
         self.connected = False
+        self._last_update_time = NEVER_TIME
+        self._debounce_cancel: CALLBACK_TYPE | None = None
+        self._debounced_update_job = HassJob(
+            self._async_handle_debounced_update,
+            f"LD2410 {ld2410_ble.address} BLE debounced update",
+        )
+
+    @callback
+    def _async_handle_debounced_update(self, _now: datetime) -> None:
+        """Handle debounced update."""
+        self._debounce_cancel = None
+        self.async_set_updated_data(None)
```

Comment:
Should this also set last update time.

Reply:
I think so.  I'll test with it to be sure

Reply:
Good call. works well